scripts/addFromConfig.py

```diff
@@ -176,9 +170,9 @@ def addStuff(cfg, options):
 
     # is the satellite in the DB?  If not add it
     try:
-        satellite_id = dbu.getEntry('Satellite', cfg['satellite']['satellite_name']).satellite_id
+        satellite_id = dbu.getSatelliteIDy(cfg['satellite']['satellite_name'])
         print('Found Satellite: {0} {1}'.format(satellite_id, dbu.getEntry('Satellite', satellite_id).satellite_name))
-    except (DButils.DBNoData, NoResultFound):
+    except BaseException:
```

Comment:
This is way too broad a match and could result in corrupting the database if some other error was thrown.

Reply:
I could not get that DButils.DBNoData to work properly.  It drove me nuts.  
I agree, it's too broad.

Reply:
Okay, will look into that. My guess is it's part of the getEntry bug above.

Reply:
I verified that using `getSatelliteID`, postgresql will raise `NoResultFound`. So reverting this.